modules/ztp-image-based-upgrade-share-container-directory.adoc

```diff
@@ -3,83 +3,9 @@
 
-You must apply a `MachineConfig` to both the seed and the target clusters during installation time to create a separate partition and share the `/var/lib/containers` directory between the two `ostree` stateroots that will be used during the upgrade process.
-
-[id="ztp-image-based-upgrade-shared-container-directory-acm_{context}"]
-== Sharing the container directory between `ostree` stateroots when using {rh-rhacm}
-
-When you are using {rh-rhacm}, you must apply a `MachineConfig` to both the seed and target clusters.
-
-[IMPORTANT]
-====
-You must complete this procedure at installation time.
-====
-
-.Prerequisites
-
-* Log in as a user with `cluster-admin` privileges.
-
-.Procedure
-
-. Apply a `MachineConfig` to create a separate partition.
-+
-[source,yaml]
-----
-apiVersion: machineconfiguration.openshift.io/v1
-kind: MachineConfig
-metadata:
-  labels:
-    machineconfiguration.openshift.io/role: master
-  name: 98-var-lib-containers-partitioned
-spec:
-  config:
-    ignition:
-      version: 3.2.0
-    storage:
-      disks:
-        - device: /dev/disk/by-id/wwn-<root_disk> <1>
-          partitions:
-            - label: varlibcontainers
-              startMiB: <start_of_parition> <2>
-              sizeMiB: <parition_size> <3>
-      filesystems:
-        - device: /dev/disk/by-partlabel/varlibcontainers
-          format: xfs
-          mountOptions:
-            - defaults
-            - prjquota
-          path: /var/lib/containers
-          wipeFilesystem: true
-    systemd:
-      units:
-        - contents: |-
-            # Generated by Butane
-            [Unit]
-            Before=local-fs.target
-            Requires=systemd-fsck@dev-disk-by\x2dpartlabel-varlibcontainers.service
-            After=systemd-fsck@dev-disk-by\x2dpartlabel-varlibcontainers.service
-
-            [Mount]
-            Where=/var/lib/containers
-            What=/dev/disk/by-partlabel/varlibcontainers
-            Type=xfs
-            Options=defaults,prjquota
-
-            [Install]
-            RequiredBy=local-fs.target
-          enabled: true
-          name: var-lib-containers.mount
-----
-<1> Specify the root disk.
-<2> Specify the start of the partition in MiB. If the value is too small, the installation will fail.
-<3> Specify the size of the partition. If the value is too small, the deployments after installation will fail.
-
-[id="ztp-image-based-upgrade-shared-container-directory-ztp_{context}"]
-== Sharing the container directory between `ostree` stateroots when using GitOps ZTP
-
-When you are using the GitOps ZTP workflow, you can do the following procedure to create a separate disk partition on both the seed and target cluster and to share the `/var/lib/containers` directory.
+When you are using the GitOps ZTP workflow, you do the following procedure to create a separate disk partition on both the seed and target cluster and to share the `/var/lib/containers` directory.
```

Comment:
I think we missing the last bit on how to use this withj Siteconfig? 

so 
1. Create butane (we have this already below)
2. Generate ignition
3. Copy that generated ignition into Siteconfig node level 

Missing 2 and 3 are documented [here](https://github.com/openshift-kni/cnf-features-deploy/blob/master/ztp/gitops-subscriptions/argocd/DiskPartitionContainers.md#convert-butane-to-ignition-and-use-it-with-siteconfigs-ignitionconfigoverride-recommended)